src/renderers/art/__tests__/ReactART-test.js

```diff
@@ -13,9 +13,6 @@
 
 'use strict';
 
-jest
-  .unmock('ReactART');
```

Comment:
We manually call unmock on a bunch of file even though automocking is off for the repo. There's now a warning in Jest for that :)